website/docs/migration/migrating-to-ferretdb.md

```diff
@@ -0,0 +1,66 @@
+---
+sidebar_position: 2
+---
+
+# Migrate from MongoDB to FerretDB
```

Comment:
<img width="239" alt="CleanShot 2023-09-20 at 12 19 38@2x" src="https://github.com/FerretDB/FerretDB/assets/11512/ec887344-d080-4cf6-87cd-2819e4776de0">

So we already have "FerretDB" in the category name (and, after all, all our docs are about FerretDB), so I would rename that page

```suggestion
# Migrate from MongoDB
```